client.go

```diff
@@ -1283,3 +1286,14 @@ type nopCloserClient struct {
 func (ncc *nopCloserClient) Close() error {
 	return nil
 }
+
+func (client *client) ParttionNotReadable(topic string, partition int32) bool {
+	pm := client.metadata[topic][partition]
+	if pm == nil {
+		return true
+	}
+	if pm.Leader == -1 {
+		return true
+	}
+	return false
```

Comment:
[simplify] We can simply return the bool conditional itself, rather than using an `if cond { return true } return false`:
```suggestion
	return pm.Leader == -1
```